src/main/java/onl/netfishers/netshot/aaa/Tacacs.java

```diff
@@ -143,19 +149,19 @@ public static UiUser authenticate(String username, String password, NetworkAddre
 				);
 				if (authoReply.isOK()) {
 					int level = UiUser.LEVEL_READONLY;
-					String role = authoReply.getValue("role");
+					String role = authoReply.getValue(roleAttribute);
 					aaaLogger.debug(MarkerFactory.getMarker("AAA"), "The TACACS+ server returned role: {}", role);
-					switch (role) {
-					case "admin":
+
+					if(role == null) {
```

Comment:
Could you fix formatting to follow rest of project's practices?
They are described in .settings/org.eclipse.jdt.core.prefs but in summary: space after `if`, CR before `else`...